### PR TITLE
Update build directives

### PR DESCRIPTION
**Pull request by robinbb**

### CODE REVIEW

File: server/shell.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  buildInputs = [
+    pkgs.cargo-make
+    pkgs.clippy
+    pkgs.openssl
+    pkgs.pkg-config
+    pkgs.podman
+    pkgs.postgresql
+    pkgs.skopeo
+
+    # Not on Darwin:
```
Comment (jenr24G) [inline server/shell.nix:33]: you can use `++ lib.optionals !stdenv.isDarwin [ #packages not to use on mac#]` to automate this!

Comment (jstavh) [reply server/shell.nix:33]: @jenr24G -- nice suggestion. Though does this depend on whether we want a static or dynamic approach to package exclusion?

**Review by jstavh — COMMENTED** [review]
It looks good. Though unable to build on my end so diagnosing if this is a local error or not.

File: server/shell.nix (hunk 1 above)
Comment (robinbb) [reply server/shell.nix:33]: I'll put this in the next version, Jen! Thanks for the tip!

**Comment by robinbb** [comment]
@jenr24G @jstavh You both looked at this PR, but neither of you gave a passing review, so it cannot merge. If it does not pass muster, then please say what must change for it to win your approval.

**Review by jenr24G — APPROVED** [review]

**Review by jstavh — COMMENTED** [review]
I'm not able to build it locally just yet. So would be good to make sure of this before merging changes regarding the .env and docker sym links to podman (Nix) now and running on different machines / OS.

**Pull request merged**